### PR TITLE
User comments

### PR DESCRIPTION
**Pull request by lxcyqx**
Add in user authentication, hide comment box if user is not logged in

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/AuthenticationServlet.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html");
+    HashMap<String, String> responseMap = new HashMap<>();
+
+    UserService userService = UserServiceFactory.getUserService();
+    if (userService.isUserLoggedIn()) {
+      String userEmail = userService.getCurrentUser().getEmail();
+      String urlToRedirectToAfterUserLogsOut = "/videos.html";
```
[the remark this answers is not in the document]
Comment (lxcyqx) [reply portfolio/src/main/java/com/google/sps/servlets/AuthenticationServlet.java]: It won't let me add static final in front of any of these but just using final works

[the remark this answers is not in the document]
Comment (lxcyqx) [reply portfolio/src/main/java/com/google/sps/servlets/AuthenticationServlet.java]: It won't let me add static final in front of any of these but just using final works

**Comment by lxcyqx** [comment]
https://lucyqu-step.ue.r.appspot.com/videos.html#comment-box

File: portfolio/src/main/java/com/google/sps/servlets/AuthenticationServlet.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (nicorod-google) [reply portfolio/src/main/java/com/google/sps/servlets/AuthenticationServlet.java]: Okay that's fine, thanks!

**Pull request merged**